src/web_interface/templates/database/database_browse.html

```diff
@@ -73,7 +73,7 @@ <h3 class="mb-3">Browse Firmware Database</h3>
 <div class="row justify-content-center">
     <div class="mt-4 col-xl-6 col-lg-8 col-md-10">
         {% if firmware_list %}
-        <ul class="list-group">
```

Comment:
there is still a (broken) border around the "Show Search Query" element
class `border-0` seems to fix the lower part but not the button
a style `border: 0;` seems to do the trick, though

Reply:
New version checked in, including fix of quotation